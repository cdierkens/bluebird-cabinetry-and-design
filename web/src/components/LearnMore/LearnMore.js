import React from "react";
import Button from "../Button";
import * as styles from "./LearnMore.module.css";

const LearnMore = () => {
  return (
    <div className={styles.LearnMoreContainer}>
      <span className={styles.LearnMoreText}>Interested in learning more?</span>
      <Button variant="blue" to="/contact" className={styles.LearnMoreLink}>
        Contact Us
      </Button>
    </div>
  );
};

export default LearnMore;
