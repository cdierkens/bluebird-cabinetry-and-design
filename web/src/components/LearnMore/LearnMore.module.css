.LearnMoreContainer {
  @apply text-center m-auto;
}

.LearnMoreText {
  @apply font-sans font-bold text-blue-dark text-xl my-9 mx-5 inline-block;
}

.LearnMoreLink {
  @apply mb-5 mx-5;
}
