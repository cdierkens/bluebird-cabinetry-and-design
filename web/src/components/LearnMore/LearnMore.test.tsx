import { render } from "@testing-library/react";
import React from "react";
import LearnMore from "./LearnMore";

it("matches snapshot", () => {
  const { asFragment } = render(<LearnMore />);

  expect(asFragment()).toMatchSnapshot();
});
