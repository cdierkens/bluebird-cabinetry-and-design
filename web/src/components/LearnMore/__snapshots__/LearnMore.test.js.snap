// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="LearnMoreContainer"
  >
    <span
      class="LearnMoreText"
    >
      Interested in learning more?
    </span>
    <a
      class="ButtonBlue LearnMoreLink focus:outline-none focus:shadow-outline"
      href="/contact"
    >
      Contact Us
    </a>
  </div>
</DocumentFragment>
`;
